docs/developer_documentation/intellij.md

```diff
@@ -219,3 +219,15 @@ Now, whenever you make changes to one or more JavaScript files, the correspondin
 the npm watch task. Once it's built, tab out of IntelliJ to trigger a refresh of the exploded war.
 As soon as IntelliJ loses focus, you'll see a task progress bar in the bottom right corner of IntelliJ.
 Wait for that task to finish and then refresh the browser. The changes will be loaded.
+
+## Speed up building the project by skipping the javascript build
+
+Building the project can be sped up by not building the javascript. To skip building the javascript use add:
+
+`-Dskip.js=true`
```

Comment:
What is the difference with using the profile ```skip-js-compile```?

Reply:
none , `skip.js=true` set the flag , that triggers the profile